### PR TITLE
IPlyska - Task 1

### PR DESCRIPTION
**Pull request by StrangerM**
First nine tasks

### CODE REVIEW

File: Bootcamp/YuriyPlLect1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                Console.WriteLine("Please input only integer. Try again.");
+                return;
+            }
+            if (radius <= 0)
+            {
+                Console.WriteLine("Radius can not be less or equal 0. Try again.");
+                return;
+            }
+
+            Console.WriteLine("Area of circle is" + 3.14 * radius * radius + "and Perimeter is" + 2 * 2.14 * radius);
```
Comment (olehkrut) [inline Bootcamp/YuriyPlLect1/Program.cs:194]: how come PI for calculating a perimeter is different from finding an area?)) 
Just use `Math.PI` so you don't have to worry about such mistakes

File: Bootcamp/YuriyPlLect1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        /// <param name="value"></param>
+        /// <returns></returns>
+        static bool IsStartFrom(string value)
+{
+            if (string.IsNullOrEmpty(value))
+            {
+                Console.WriteLine("Value can not be an empty");
+                return false;
+            }
+            return value.Trim().Take(2).ToArray().ToString() == "C#";
```
Comment (olehkrut) [inline Bootcamp/YuriyPlLect1/Program.cs:171]: no need in fancy LINQ when you have string method `StartsWith`

File: Bootcamp/YuriyPlLect1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            var secondNumber = Console.ReadLine().Trim();
+            int secondValue;
+
+            if (!int.TryParse(secondNumber, out secondValue))
+            {
+                Console.WriteLine("Please input only integer. Try again.");
+                return;
+            }
+
+            var result = firstValue == secondValue ? firstValue + secondValue : (firstValue + secondValue) / 8;
```
Comment (olehkrut) [inline Bootcamp/YuriyPlLect1/Program.cs:108]: `int / int` gives you int, that's not what you usually expect, use `8.0` so the overall result will be double.
e.g. 12 / 8 = 1

**Review by olehkrut — COMMENTED** [review]
The code is clean which is good, well-done.
Small mistakes are present, but easy to fix.
Not all tasks are done, but it's not that critical

File: Bootcamp/YuriyPlLect1/Program.cs (hunk 3 above)
Comment (StrangerM) [reply Bootcamp/YuriyPlLect1/Program.cs:108]: changed to double

File: Bootcamp/YuriyPlLect1/Program.cs (hunk 2 above)
Comment (StrangerM) [reply Bootcamp/YuriyPlLect1/Program.cs:171]: changed to StartsWith

File: Bootcamp/YuriyPlLect1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+            {
+                Console.WriteLine("Value can not be an empty");
+                return;
+            }
+            if (text.Length < 5)
+            {
+                Console.WriteLine(text);
+            }
+
+            Console.WriteLine(new string(text.Take(5).ToArray()).ToUpper() + new string(text.Skip(5).ToArray()).ToString());
```
[the remark this answers is not in the document]
Comment (StrangerM) [reply Bootcamp/YuriyPlLect1/Program.cs:146]: changed to Join

File: Bootcamp/YuriyPlLect1/Program.cs (hunk 1 above)
Comment (StrangerM) [reply Bootcamp/YuriyPlLect1/Program.cs:194]: Changed to Math.PI

File: Bootcamp/YuriyPlLect1/Program.cs [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+        /// <summary>
+        /// multiplication of three numbers
+        /// </summary>
+        static void MultipleNumbers()
+        {
+            Console.WriteLine("Please input first number");
+            var firstNumber = Console.ReadLine().Trim();
+            int firstValue;
+
+           if (!int.TryParse(firstNumber, out firstValue))
```
[the remark this answers is not in the document]
Comment (StrangerM) [reply Bootcamp/YuriyPlLect1/Program.cs:36]: added local function

**Review by olehkrut — APPROVED** [review]

**Pull request merged**